python-package/lets_plot/plot/coord.py

```diff
@@ -15,20 +15,34 @@
 
 def coord_cartesian(xlim=None, ylim=None):
     """
-    The Cartesian coordinate system is the most familiar, and common, type of coordinate system.
-    Setting limits on the coordinate system will zoom the plot (like you're looking at it with a magnifying glass),
-    and will not change the underlying data like setting limits on a scale will.
+    The Cartesian coordinate system is the most familiar and common type of coordinate system.
+    Setting limits on the coordinate system will zoom the plot like you're looking at it with a magnifying glass.
+    It does not change the underlying data as setting limits on a scale does.
 
     Parameters
     ----------
-    xlim : list of numbers (2 elements)
-           Limits for the x axes.
-           1st element defines lower limit, 2nd element defines upper limit.
-           `None` means no lower / upper bound - depending on the index in list.
-    ylim : list of numbers (2 elements)
-           Limits for the y axes.
-           1st element defines lower limit, 2nd element defines upper limit.
-           `None` means no lower / upper bound - depending on the index in list.
+    xlim : list
+        Limits (2 elements) for the x axis.
+        1st element defines lower limit, 2nd element defines upper limit.
+        None means no lower / upper bound - depending on the index in list.
+    ylim : list
+        Limits (2 elements) for the y axis.
+        1st element defines lower limit, 2nd element defines upper limit.
+        None means no lower / upper bound - depending on the index in list.
+
```

Comment:
Why "Returns" isn't specified? Spolier: it is of `FeatureSpec` type, according to code.